### DISCUSSION TITLE
Raspberry Pi Pico W

### DISCUSSION THREAD

**Discussion by mtoko** [phoddie/node-red-mcu#10] [General]
The Raspberry Pi Pico W (wireless) was available for purchase yesterday. I have not had much time to investigate, but I assume running Node Red on a Pico with an MQTT flow could be possible

**Comment by phoddie** [comment]
It is available for purchase, and sold-out everywhere! I ordered a couple, for whenever they are back in stock.

The Moddable SDK runs on Pico so it should be possible to run flows there too. There would be some work to get the needed Moddable SDK modules implemented for Pico, but that's not too bad.
